less/admin/AdminNav.less

```diff
@@ -107,22 +104,31 @@
         margin-bottom: 20px;
       }
 
+      .item-category-core {
+        > .ExtensionListTitle {
+          margin-top: 10px;
+        }
+      }
+
       > li {
         > a {
           padding: 10px 10px 10px 45px;
           display: block;
           text-decoration: none;
         }
+
         > a,
         > a:hover,
         &.active > a {
           color: @text-color;
         }
+
         > a:hover {
           background: @control-bg;
         }
+
         &.active > a {
-          background: @primary-color;
+          background: @muted-color;
```

Comment:
this is technically a nitpick but a good one, can we use `@control-color`, they're the same colors, but best to use `@control-color` because that's the correct purpose here (so that in the future we are not restricted if we edit muted color)

Reply:
also, below this, can we move the `color: @body-bg` up from `.Button-label, .Button-icon` up here to the anchor, it's better and also fixes a problem with the BC extension page item when it's active. (I think I posted an image in a comment above)